content/publication/batman.md

```diff
@@ -0,0 +1,27 @@
++++
+abstract = "Active Inference is an emerging framework for designing intelligent agents. In an Active Inference setting, any task is formulated as a variational free energy minimisation problem on a generative probabilistic model. Goal-directed behaviour relies on a clear specification of desired future observations. Learning desired observations would open up the Active Inference approach to problems where these are difficult to specify a priori. This paper introduces the BAyesian Target Modelling for Active iNference (BATMAN) approach, which augments an Active Inference agent with an additional, separate model that learns desired future observations from a separate data source. The main contribution of this paper is the design of a coupled generative model structure that facilitates learning desired future observations for Active Inference agents and supports integration of Active Inference and classical methods in a joint framework. We provide proof-of-concept validation for BATMAN through simulations."
+abstract_short = ""
+date = "2020-05-07T18:30:00+02:00"
+image = ""
+image_preview = ""
+math = false
+publication = "ICASSP 2020 - 2020 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)"
+publication_short = ""
```

Comment:
Thanks for PR @MagnusKoudahl 
Please erase "ICASSP 2020" from publication parameter and add it to publication short.

Reply:
:+1: